File: Excepciones2/LanzadorExcepcion.java
public class LanzadorExcepcion {

    private static boolean verificarElegibilidad(int edad, int peso) {
        if (edad < 15 || peso < 50) {
            throw new ArithmeticException("La persona no puede usar el juego.");
        }
        return true;
    }

    public static void main(String[] args) {
        boolean elegible = false;
        System.out.println("¡Bienvenido al servicio de registro!");
        try {
            elegible = verificarElegibilidad(Integer.valueOf(args[0]),
                Integer.valueOf(args[1]));
        } catch (ArithmeticException ex) {
            System.out.println(ex.getMessage());
        } finally {
            if (elegible) {
                System.out.println("La persona cumple con los criterios.");
            }
        }
    }
}

File: Excepciones2/lanzador_excepcion.py
import sys

class LanzadorExcepcion:

    def __verificarElegibilidad(edad, peso):
        if (edad < 15 or peso < 50):
            raise ArithmeticError("La persona no puede usar el juego.")
        return (True)


    def main(edad, peso):
        elegible = False
        print("¡Bienvenido al servicio de registro!")
        try:
            elegible = LanzadorExcepcion.__verificarElegibilidad(edad, peso)
        except ArithmeticError as ex:
            print(ex);
        finally:
            if (elegible):
                print("El cliente cumple con los criterios.")


LanzadorExcepcion.main(int(sys.argv[1]), int(sys.argv[2]))
